tools/reproduce_comparison/compareLinux.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+[ $# -lt 1 ] && echo Usage: $0 SBOMURL && exit 1
+SBOMURL=$1
+#apt install gcc     g++ make autoconf file unzip zip libasound2-dev libcups2-dev libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev
+if test -r /etc/redhat-release; then
```

Comment:
two if blocks could be turned into functions?

Reply:
I've done this, although I honestly don't think it really aids the readability of a script of this size. Also it restricts which shell it can run on since the first line is currently `/bin/sh`  (Admittedly not a problem on RH/CentOS Linux though, which is what script is designed for!)